src/infrastructure/github.ts

```diff
@@ -6,9 +6,9 @@ import { GitHub as ActionsGitHub } from '@actions/github/lib/utils'
 
 export type Octokit = InstanceType<typeof ActionsGitHub>
 export type PullsGetResponse =
-  Endpoints['GET /repos/{owner}/{repo}/pulls/{pull_number}']['response']
-export type PullsGetAssociatedWithCommitResponse =
-  Endpoints['GET /repos/{owner}/{repo}/commits/{commit_sha}/pulls']['response']
+  Endpoints['GET /repos/{owner}/{repo}/pulls/{pull_number}']['response']['data']
+export type GetPullRequestsAssociatedWithCommitResponse =
```

Comment:
Why using a verb to name an object? `PullRequestsAssociatedWithCommit` should be enough, no?